lagom-eventstore-example/README.md

```diff
@@ -13,7 +13,7 @@ To build and run this example, you need:
 - [git](https://git-scm.com/)
 - [Java SE 8 JDK](http://www.oracle.com/technetwork/java/javase/overview/index.html)
 - [Maven 3.2.1+](https://maven.apache.org/) to build and run the Lagom project (3.5.0 recommended)
-- IBM Db2 Event Store [downloaded](https://github.com/IBMProjectEventStore/EventStore-DeveloperPreview/releases/tag/1.1.1), [installed](https://www.ibm.com/support/knowledgecenter/SSGNPV/eventstore/desktop/welcome.html) and running on your local machine. (This process requires 12+Gb of downloads)
+- IBM Project EventStore [downloaded](https://github.com/IBMProjectEventStore/EventStore-DeveloperEdition/releases), [installed](https://www.ibm.com/support/knowledgecenter/SSGNPV/eventstore/desktop/welcome.html) and running on your local machine. (This process requires 12+Gb of downloads)
```

Comment:
This link label was changed from "IBM Project EventStore" to "IBM Db2 Event Store" by @stevemart in #31. Did you intend to change it back or was that a merge conflict?

Reply:
@markstur maaaaark hai.

@TimMoore probably a merge conflict, i think.

Reply:
merge mistake